### ISSUE TITLE
Page About

### ISSUE THREAD

**Issue by Wilker-a** [rkkr/simple-keyboard#131]
a simple page in the settings for quick info about the author, contribuitors, license, and links to the source code and other places.

**Comment by JM480** [comment]
I think this could be useful for making it easier for people to report bugs and request new features.

**Comment by inson1** [comment]
I wanted create same issue. 
Plus also current version

**Comment by rkkr** [comment]
Privacy policy and license are already there. Source code could be added, theu they all are just github links.

**Comment by inson1** [comment]
@rkkr yes, but I meant in About page. And remove them from main menu. So main manu can stay minimal.

**Comment by Cwpute** [comment]
The licence and Privacy Policy should probably not be links, as one could download your app and use it offline. Also in the event Github becomes available (highly unpredictable i know) both documents would be unreachable or lost.
